tests/appsec/iast/test_iast_path_traversal.py

```diff
@@ -0,0 +1,34 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released
+
+# Weblog are ok for nodejs/express4 and java/spring-boot
+@coverage.basic
+@released(
+    dotnet="?",
+    golang="?",
+    java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.108.0", "spring-boot-openliberty": "0.108.0", "*": "?"},
```

Comment:
I don't think we've released this in java! Let's change it to `java=?`.

Reply:
Done